### ISSUE TITLE
new term: Early-onset of dilatation of the cerebral artery

### ISSUE THREAD

**Issue by albodrug** [obophenotype/human-phenotype-ontology#10979]
**Preferred term label:**
Early-onset of dilatation of the cerebral artery

**Synonyms**
Early-onset of intracranial aneurysm, Early-onset of cerebral aneurysm

**Definition (free text, please give PubMed ID)**
PMID: 38315279 "Patients aged < 50 years with a family history of intracranial aneurysm or subarachnoid hemorrhage or with CKD stages 4–5 may be at an increased risk of early-onset intracranial aneurysm."

PMID: 29930801
"We thus defined patients aged ≤65 years as early-onset cases in the present study."

**Parent term (use hpo.jax.org/app)**
Dilatation of the cerebral artery HP:0004944

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
could characterize:
MONDO:0005291 A congenital or acquired aneurysm within the cranium.
ORPHA:231160 Familial cerebral saccular aneurysm 

**Your nano-attribution (ORCID)**
0009-0006-3873-102X

**Comment by pnrobinson** [comment]
@albodrug -- We would not want to create HPO terms with "early-onset", because this would mean that we would need to create hundreds or thousands of additional terms. Instead, the HPO approach would be to use an Dilatation of the cerebral artery HP:0004944 with an indication of Onset, ideally in combination with a phenopacket - https://obophenotype.github.io/human-phenotype-ontology/phenopackets/
